### DISCUSSION TITLE
Update to Juggernaut XL v8?

### DISCUSSION THREAD

**Discussion by petergerdy** [lllyasviel/Fooocus#2121] [General]
Hey,

i just saw that there is a version 8 of the Juggernaut model, how could I easily update to v8? Just replace in config.py?

I was also wondering about the license for this model, is this also completely free like Diffusion XL?

Love Fooocus! Keep it up! P.

**Comment by SGThirkell** [comment]
Hi, add  --always-download-new-model to your run.bat and it will always look for new versions.
So it'll look like this;

.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-download-new-model
pause

**Comment by NaschBear** [comment]
Hi 

i have a question about the Fooocus Updates. 

I think I read that there is a command that allows Fooocus to always pull new updates, is that true?

**Reply by mashb1t** [reply]
This is the default, yes. Fooocus automatically updates to the latest version. You can find the output of the update as the first line in your run.bat or startup command in general when using entry_with_update.py.

**Reply by NaschBear** [reply]
ah okay, I was wondering why you should enter a command again when fooocus does it independently? That just confused me, thank you very much Herr Schmid
